src/main/java/org/savantbuild/dep/domain/ReifiedArtifact.java

```diff
@@ -41,6 +41,10 @@ public ReifiedArtifact(ArtifactID id, Version version, List<License> licenses) {
     this(id, version, null, Collections.emptyList(), licenses);
   }
 
+  public ReifiedArtifact(ArtifactID id, Version version, List<License> licenses, String nonSemanticVersion) {
```

Comment:
I tend to keep all constructors and methods ordered the same. Since this calls another constructor that has `nonSemanticVersion` as the 3rd parameter, I'd update this one to match. This reduces the mental overhead of what each value is for the constructors.

Reply:
Sure. updated.